Diary/src/main/java/com/microsoft/device/display/samples/diary/ui/theme/Theme.kt

```diff
@@ -0,0 +1,48 @@
+package com.microsoft.device.display.samples.diary.ui.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+
+private val DarkColorPalette = darkColors(
+    primary = Purple200,
+    primaryVariant = Purple700,
+    secondary = Teal200,
+    secondaryVariant = Teal500
+)
```

Comment:
If we don't end up using the dark theme, you can delete this and the `darkTheme` parameter below